.github/workflows/build-wheels.yml

```diff
@@ -82,7 +82,7 @@ jobs:
         - os: ubuntu-latest
           manylinux-version: 2010
           python-version: 3.8
-          build-depends: numpy==1.17.3
+          build-depends: numpy==1.18.5
```

Comment:
Why this change? For backwards compatibility, we want to build against the _oldest_ possible numpy version that has wheels for Python 3.8. I can see that:

- [1.17.3](https://pypi.org/project/numpy/1.17.3/#files): *does* have wheels for Py3.8
- [1.17.2](https://pypi.org/project/numpy/1.17.3/#files): *does not* have wheels for Py3.8

So isn't 1.17.3 the right choice here?